awx/main/conf.py

```diff
@@ -345,6 +346,61 @@ def _load_default_license_from_file():
     category_slug='jobs',
 )
 
+register(
+    'AWX_RESOURCE_PROFILING_ENABLED',
+    field_class=fields.BooleanField,
+    default=False,
+    label=_('Enable resource profiling on all tower jobs'),
+    help_text=_('If set, resource profiling data will be collected on all jobs.'),  # noqa
+    category=_('Jobs'),
+    category_slug='jobs',
+)
+
+register(
+    'AWX_RESOURCE_PROFILING_BASE_CGROUP',
+    field_class=fields.CharField,
+    allow_blank=False,
+    default='ansible-runner',
```

Comment:
What's the purpose of this being a knob?

Reply:
Good question - having gone through the whole feature, not sure that I _can_ think of much of a reason why we would want this. When I was first adding options, this seemed like something we might want. Removing.